docs/releases/creating-a-release.md

```diff
@@ -94,7 +94,7 @@ Both beta and stable releases follow a two-step workflow, aided by GitHub action
 
 ### Step 1.1: Create a Pre-release Branch
 
-We want to create a pre-release branch because this branch will act as a place to perform actions like bumping package versions, grooming the changelog, and performing string translations. This branch will be the place that the release branch is created from before it is merged back into main.
+We want to create a pre-release branch because this branch will act as a place to perform actions like bumping package versions, grooming the changelog, and performing string translations. This branch will be the place that the release branch is created from before it is merged back into main. **Note:** Be sure to look at the `package.json` of the `communication-react` package to determine which bump type you will need for your release.
```

Comment:
> to determine which bump type you will need for your release

Pre-release branches are typically off the `main` branch meaning the package.json will always look like `x.y.z-beta.n` (the package.json isn't altered until the release-branch action). So I don't believe looking at the package.json will tell you the bump type will it?
To determine bump type the releaser would need to evaluate:
- Is this a Stable or a Beta release
- Is this a patch or minor version bump

Reply:
it will, since we need to be looking at the version that is in the package.json at the time of the release to have the correct bump.

if for example we just did a beta release like `1.17.0`, and we are wanting to go to `1.18.0-beta.1` we would normally think that the bump type would be `beta-minor`. In this case however to achieve the correct version we want to use the `beta-release-beta` workflow to bump the beta. 

This is because the `package.json` in `communication-react` is going to be `1.18.0-beta.0` following the `1.17.0` release. if the case is that the release driver uses `beta-minor` release type the resulting version would be `1.19.0-beta.1` which is incorrect as we have skipped `1.18.0-beta.1`